### PR TITLE
Bug Fixes to enable multi chain login to existing accounts

### PR DESCRIPTION
**Pull request by NightTrek**

### CODE REVIEW

File: src/hooks/account/UseCreateFun.ts [hunk 1]
```diff
@@ -104,40 +104,39 @@ export const useCreateFun = () => {
...
               setNewAccountUsers(allUsers, allUsers[0])
             })
             .catch()
-
-          setLogin(newAccountAddress, newFunWallet)
+          setLogin(account, newFunWallet)
           return newFunWallet
-        } else if (args.uniqueId) {
+        } else if (args.users) {
+          const WALLET_UNIQUE_ID = await auth.getWalletUniqueId(args.index ?? 0)
```
Comment (Chazzzzzzz) [inline src/hooks/account/UseCreateFun.ts:122]: const walletUniqueId = xx


full capital is for constant value

File: src/hooks/account/UseCreateFun.ts [hunk 2]
```diff
@@ -104,40 +104,39 @@ export const useCreateFun = () => {
             .catch()
           setLogin(account, newFunWallet)
           return newFunWallet
```
Comment (Chazzzzzzz) [inline src/hooks/account/UseCreateFun.ts:106]: not sure this is right for parsing in "Address"

File: src/hooks/account/UseCreateFun.ts [hunk 3]
```diff
@@ -104,40 +104,39 @@ export const useCreateFun = () => {
...
             .then((allUsers) => {
               setNewAccountUsers(allUsers, allUsers[0])
             })
             .catch()
-          setLogin(account, newFunWallet)
+
+          setLogin(newAccountAddress, newFunWallet)
           return newFunWallet
         } else {
           const WALLET_UNIQUE_ID = await auth.getWalletUniqueId(args.index ?? 0)
```
Comment (Chazzzzzzz) [inline src/hooks/account/UseCreateFun.ts:142]: same here

File: src/hooks/account/UseCreateFun.ts [hunk 4]
```diff
@@ -104,40 +104,39 @@ export const useCreateFun = () => {
...
+        } else if (args.users) {
+          const WALLET_UNIQUE_ID = await auth.getWalletUniqueId(args.index ?? 0)
           const newFunWallet = new FunWallet({
-            users: [{ userId: await auth.getUserId() }],
-            uniqueId: args.uniqueId,
+            users: args.users,
+            uniqueId: WALLET_UNIQUE_ID,
           })
-          const account = await newFunWallet.getAddress()
+          const newGroupIds = args.users.map((user) => {
```
Comment (Chazzzzzzz) [inline src/hooks/account/UseCreateFun.ts:127]: user may not be groups

**Review by Chazzzzzzz — APPROVED** [review]

**Review by ken-fun — APPROVED** [review]

**Pull request merged**